extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunner.java

```diff
@@ -346,7 +348,8 @@ private List<String> generateCommand(Task task)
   {
     final List<String> command = new ArrayList<>();
     command.add("/peon.sh");
-    command.add(taskConfig.getTaskDir(task.getId()).toString());
+    command.add(dirTracker.getBaseTaskDir(task.getId()).getAbsolutePath());
+    command.add(task.getId());
```

Comment:
what's the reason for adding `taskId` to the command?

Reply:
The command originally needed the task's working directory.
It now requires the base directory for TaskStorageDirTracker. The taskId along with the base directory can be used to obtain the working directory